mmv1/products/netapp/volume.yaml

```diff
@@ -0,0 +1,416 @@
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Quickstart': 'https://cloud.google.com/netapp/volumes/docs/get-started/quickstarts/create-volume'
+    'Documentation': 'https://cloud.google.com/netapp/volumes/docs/configure-and-use/volumes/overview'
+  api: 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.volumes'
+base_url: projects/{{project}}/locations/{{location}}/volumes
+self_link: projects/{{project}}/locations/{{location}}/volumes/{{name}}
+create_url: projects/{{project}}/locations/{{location}}/volumes?volumeId={{name}}
+update_url: projects/{{project}}/locations/{{location}}/volumes/{{name}}
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+id_format: 'projects/{{project}}/locations/{{location}}/volumes/{{name}}'
+import_format: ['projects/{{project}}/locations/{{location}}/volumes/{{name}}']
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Name of the pool location. Usually a region name, expect for some STANDARD service level pools which require a zone name.
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description:
+      The name of the volume. Needs to be unique per location.
+    required: true
+    immutable: true
+    url_param_only: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'volume_basic'
+    primary_resource_id: 'test_volume'
+    vars:
+      volume_name: 'test-volume'
+      pool_name: 'test-pool'
+      network_name: 'test-network'
+    test_vars_overrides:
+      network_name: 'acctest.BootstrapSharedServiceNetworkingConnection(t, "gcnv-network-config-1", acctest.ServiceNetworkWithParentService("netapp.servicenetworking.goog"))'
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'shareName'
+    description: |
+      Share name (SMB) or export path (NFS) of the volume. Needs to be unique per location.
+    required: true
+    immutable: true
+  - !ruby/object:Api::Type::String
+    name: 'psaRange'
+    description: |
+      Name of the Private Service Access allocated range. Inherited from storage pool.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'storagePool'
+    description: |
+      Name of the storage pool to create the volume in. Pool needs enough spare capacity to accomodate the volume.
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'network'
+    description: |
+      VPC network name with format: `projects/{{project}}/global/networks/{{network}}`. Inherited from storage pool.
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'serviceLevel'
+    description: |
+      Service level of the volume. Inherited from storage pool.
+    values:
+      - :SERVICE_LEVEL_UNSPECIFIED
```

Comment:
_UNSPECIFIED is generally not useful for users to send, we should probably remove this unless there is a reason someone would use this

Reply:
This is an output:true field. The user cannot send the value.